docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,43 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `static`, `dynamic`
+- **Default value:** `static`
+
+Trino reads catalog property files and configures available catalogs on server
+startup. When set to `dynamic`, catalog configuration can also be managed using
+[](/sql/create-catalog) and [](/sql/drop-catalog). Creating and dropping
+catalogs using these SQL commands will create new and delete existing catalog
+property files on all nodes.
+
+:::{warning}
+This feature is experimental only. Because of the security implications the
+syntax might change and be backward incompatible.
+:::
+
+:::{warning}
+Some connectors are known not to release all resources when dropping a catalog
+that uses such connector. This includes all connectors that can read data from
+HDFS, S3, GCS, or Azure, which are {doc}`/connector/hive`,
+{doc}`/connector/iceberg`, {doc}`/connector/delta-lake`, and
+{doc}`/connector/hudi`.
+:::
+
+:::{warning}
+The complete CREATE CATALOG query can be logged, and visible in the {doc}`Web
+UI </admin/web-interface>`. This includes any sensitive properties, like
+passwords and other credentials. See {doc}`secrets </security/secrets>`.
+:::
+
+## `catalog.prune.update-interval`
+
+- **Type:** {ref}`prop-type-duration`
+- **Default value:** `5s`
+- **Minimum value:** `1s`
+
+Interval for pruning dropped catalogs. Dropping a catalog does not interrupt
```

Comment:
we should add info on where they are dropped .. on coordinator and all workers..

Reply:
This is not relevant here, but I explained that in the description of other properties.